Don't find all dylibs on macOS.

The purpose of this code was originally to get a list of linked libraries to
copy them for a self-contained installation. We don't want that in MacPorts, and
the code that copied the libraries using these variables was later removed:

https://github.com/zorba-processor/zorba/commit/af2466707086db95e1f634178b2ad8665c004035

This code also caused an endless loop if any of the libraries weren't found by
CMake, for example when we deliberately tell it not to find Xerces-C.
--- CMakeLists.txt.orig	2015-06-16 06:26:38.000000000 -0500
+++ CMakeLists.txt	2024-11-15 00:04:43.000000000 -0600
@@ -464,26 +464,6 @@
 ADD_DEFINITIONS(${DEFINITIONS})
 
 
-################################################################################
-# find all dylib files (i.e. transitive closure of the library files
-# that are linked)
-IF(APPLE)
-  EXECUTE_PROCESS(
-    COMMAND ${CMAKE_SOURCE_DIR}/scripts/find_macosx_libraries.sh ${ICU_LIBRARIES}
-    OUTPUT_VARIABLE Zorba_ICUUC)
-  EXECUTE_PROCESS(
-    COMMAND ${CMAKE_SOURCE_DIR}/scripts/find_macosx_libraries.sh ${ICU_I18N_LIBRARIES}
-    OUTPUT_VARIABLE Zorba_ICUI18N)
-  EXECUTE_PROCESS(
-    COMMAND ${CMAKE_SOURCE_DIR}/scripts/find_macosx_libraries.sh ${ICU_DATA_LIBRARIES}
-    OUTPUT_VARIABLE Zorba_ICU_DATA)
-  EXECUTE_PROCESS(
-    COMMAND ${CMAKE_SOURCE_DIR}/scripts/find_macosx_libraries.sh ${XERCESC_LIBRARY}
-    OUTPUT_VARIABLE Zorba_XERCESC)
-  EXECUTE_PROCESS(
-    COMMAND ${CMAKE_SOURCE_DIR}/scripts/find_macosx_libraries.sh ${LIBXML2_LIBRARIES}
-    OUTPUT_VARIABLE Zorba_XML2)
-ENDIF(APPLE)
 
 
 SET (ZORBA_MODULES_DIR "${CMAKE_SOURCE_DIR}/../zorba_modules"
